qiling/os/windows/dlls/user32.py

```diff
@@ -113,7 +113,11 @@ def hook_CloseClipboard(ql, address, params):
     "hMem": STRING
 })
 def hook_SetClipboardData(ql, address, params):
-    return ql.clipboard.set_data(params['uFormat'], bytes(params['hMem'], 'ascii'))
+    try:
+        data = bytes(params['hMem'], 'ascii')
+    except:
+        data = ""
```

Comment:
The type of `data` is inconsistent. Maybe using `data=b""` instead?

Reply:
You're right. I fixed it. Thank you.